reconstruction.py

```diff
@@ -896,12 +896,14 @@ def reconstruct(self,evrange=(-1,-1,-1)):
             file_url = sw.swift_root_file(options.tag, int(options.run))
             print ('Downloading file: ' + file_url)
             options.tmpname = sw.swift_download_root_file(file_url,int(options.run),tmpdir)
+        if options.rawdata_tier == 'h5':
+            options.tmpname = ("%s/histograms_Run%05d.h5" % (tmpdir,int(options.run)))
```

Comment:
What is this line for? This is the download part, which we do not have for h5. So Either we implement it or we just put a 
if options.rawdata_tier == 'h5': error
To use a local folder where the file already exists there is line 883